pkg/chartverifier/profiles/default.go

```diff
@@ -38,6 +39,7 @@ func getDefaultProfile(msg string) *Profile {
 		{Name: fmt.Sprintf("%s/%s", CheckVersion10, checks.NotContainCsiObjectsName), Type: checks.MandatoryCheckType},
 		{Name: fmt.Sprintf("%s/%s", CheckVersion10, checks.ImagesAreCertifiedName), Type: checks.MandatoryCheckType},
 		{Name: fmt.Sprintf("%s/%s", CheckVersion10, checks.ChartTestingName), Type: checks.MandatoryCheckType},
+		{Name: fmt.Sprintf("%s/%s", CheckVersion10, checks.NoMissingRequiredAnnotationsName), Type: checks.MandatoryCheckType},
```

Comment:
Thinking we need to maintain two defaults version 1.0 and version 1.1

Reply:
Defaulted to 1.1 PTAL